go/vt/vtgate/vstream_manager.go

```diff
@@ -544,6 +548,22 @@ func (vs *vstream) streamFromTablet(ctx context.Context, sgtid *binlogdatapb.Sha
 						return err
 					}
 
+					if err := vs.sendAll(ctx, sgtid, eventss); err != nil {
+						return err
+					}
+					eventss = nil
+					sendevents = nil
+				case binlogdatapb.VEventType_COPY_COMPLETED:
+					sendevents = append(sendevents, event)
+					if fullyCopied := vs.generateFullyCopyCompleted(ctx, sgtid, event); fullyCopied != nil {
+						sendevents = append(sendevents, fullyCopied)
```

Comment:
Only this addition is different from the case of, for example, `OTHER`.
In the case of `COPY_COMPLETED`, it's possible to generate and send an extra event.